core/ExceptionHandler.php

```diff
@@ -137,4 +144,15 @@ private static function getErrorResponse($ex)
 
         return $result;
     }
+
+    private static function logException($exception)
+    {
+        try {
+            StaticContainer::get(LoggerInterface::class)->error('Uncaught exception: {exception}', [
+                'exception' => $exception,
+            ]);
+        } catch (ContainerException $ex) {
+            // ignore (occurs if exception is thrown when resolving DI entries)
```

Comment:
lets maybe also catch `ContainerDoesNotExistException` in case the container is not set up yet?

Reply:
👍